src/js/view/styledComponents/TabPanelStyledComponents.js

```diff
@@ -45,3 +45,22 @@ export const Button = styled.div`
   padding: 5px;
   margin-left: 5px;
 `;
+
+export const Indicator = styled.div`
+  width: 10px;
+  height: 10px;
+  display: inline-block;
+  background-color: lightgreen;
```

Comment:
Perhaps this color can be changed that stands out a bit more and is more attention-grabbing, like orange? Lightgreen feels a bit too muted, at least for me personally.
```suggestion
  background-color: orange;
```

Reply:
Makes sense, will change color to something more alerting.